CHANGELOG.md.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: e0291b694a2344f4abc085ebe276fefe
+guid: 0679a03ffce774e4c888d4c1cfcefedb
```

Comment:
Seems an unnecessary meta file changes. And more in this PR.

Reply:
Changes discarded. Repository was connected as git submodule, that's the reason of meta files changed